### DISCUSSION TITLE
Wrong native symbols after upgrading to LLVM13

### DISCUSSION THREAD

**Discussion by hvanhovell** [async-profiler/async-profiler#628] [Q&A]
Hi all, 

Let me begin with saying that the async-profiler is a massively useful tool for us. It helped us to debug (subtle) performance problems countless times.

We recently moved from LLVM 9 to LLVM 13. After this the native frames captured by the async profiler use the wrong symbol names. Does anyone have any idea what could be causing this?

Cheers,
Herman

**Comment by apangin** [comment]
Hi and thank you for your interest in async-profiler.

I'm not sure I understand the question correctly. How does LLVM relate to async-profiler? What exactly do you mean by "use the wrong symbol names"? How did the profile look before and after the change? Do you have an example that demonstrates the problem?
